spec/scenarios/subscriptions/terminate_ended_spec.rb

```diff
@@ -178,6 +178,46 @@
       end
     end
 
+    context 'when ending_at is not set and subscription is terminated on the day of creation' do
+      it 'bills correctly only 1 day' do
+        subscription = nil
+
+        travel_to(creation_time) do
+          create_subscription(
+            {
+              external_customer_id: customer.external_id,
+              external_id: customer.external_id,
+              plan_code: plan.code,
+              billing_time: 'anniversary',
+              subscription_at: subscription_at.iso8601,
+              ending_at: nil,
+            },
+          )
+
+          subscription = customer.subscriptions.first
+          expect(subscription).to be_active
+        end
+
+        Organization.update_all(webhook_url: nil) # rubocop:disable Rails/SkipsModelValidationss
```

Comment:
Rails/SkipsModelValidations: Avoid using `update_all` because it skips validations. (https://guides.rubyonrails.org/active_record_validations.html#skipping-validations)